diff --git i/node/binding.gyp w/node/binding.gyp
index b0d92eae..c5e504fa 100644
--- i/node/binding.gyp
+++ w/node/binding.gyp
@@ -43,7 +43,7 @@
                   "include_dirs": ["../deltachat-ffi"],
                   "ldflags": ["-Wl,-Bsymbolic"], # Prevent sqlite3 from electron from overriding sqlcipher
                   "libraries": [
-                    "../../target/release/libdeltachat.a",
+                    "../../target/x86_64-apple-darwin/release/libdeltachat.a",
                     "-ldl",
                   ],
                   "conditions": [],
